<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutPLLForm</class>
 <widget class="QWidget" name="AboutPLLForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CubeWidget" name="cube0" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="CubeWidget" name="cube1" native="true">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="CubeWidget" name="cube2" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="CubeWidget" name="cube3" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="FourRotationsWidget" name="fourCasesOfOnePLL" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>621</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>25</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0째</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>25</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>90째</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>398</x>
      <y>25</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>180째</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>25</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>270째</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CubeWidget</class>
   <extends>QWidget</extends>
   <header>cubewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FourRotationsWidget</class>
   <extends>QWidget</extends>
   <header>fourrotationswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
